pkg/kubelet/kuberuntime/kuberuntime_manager.go

```diff
@@ -551,8 +555,12 @@ func IsInPlacePodVerticalScalingAllowed(pod *v1.Pod) bool {
 
 // computePodResizeAction determines the actions required (if any) to resize the given container.
 // Returns whether to keep (true) or restart (false) the container.
-func (m *kubeGenericRuntimeManager) computePodResizeAction(pod *v1.Pod, containerIdx int, kubeContainerStatus *kubecontainer.Status, changes *podActions) (keepContainer bool) {
+// TODO(vibansal): Make this function to be agnostic to whether it is dealing with a restartable init container or not (i.e. remove the argument `isRestartableInitContainer`).
```

Comment:
is it TODO to address in this PR or later? Merging PRs with TODO is generally not a good practice unless there is a good reason for it.

Reply:
This needs to be resolved later on. If I will not pass `isRestartableInitContainer` to the function then it will need more refactoring, and right now due to code freeze deadline coming up we have decided to do that part later on.